src/app/auth.tsx

```diff
@@ -26,8 +28,35 @@ type AuthParams = {
 
 const AuthCallback = () => {
   const params = useGlobalSearchParams<AuthParams>();
-
   const router = useRouter();
+  const isAuth = useUser((data) => data.isAuth);
+
+  useEffect(() => {
+    // Check if all necessary params are available
+    if (!params || !params._id || !params.token) {
+      return;
+    }
+
+    const user = {
+      id: params._id,
+      deviceIds: [params.device_id].filter(Boolean) as string[],
+      currentDeviceId: params.device_id || null,
+      name: params.name || "User",
```

Comment:
instead of "User" just leave an empty string

Reply:
updated